### PR TITLE
🇺🇳 [Validation]: Added lang attribute to <html> tag for all languages.

### PR DESCRIPTION
**Pull request by rohitjethoe**
### Validated html

Added a html lang attribute to the html tag in the right page language with JavaScript.

### CODE REVIEW

**Comment by baltpeter** [comment]
Hi,

thanks for the suggestion!

We will soon move to a new hosting solution that will allow us to have proper per-language 404 pages that don't need JavaScript for the translation (see #207). We will then be able to just add the `lang` attribute to the `<html>` tag directly.

Do you think this can wait until then? Otherwise, I would request some changes and propose a cleaner implementation. I'll leave that up to you. :)

**Comment by rohitjethoe** [comment]
@baltpeter There isn't any other JavaScript file linked to the 404 page so I think this is currently the cleanest implementation.

File: static/404.html [hunk 1]
```diff
@@ -81,6 +81,26 @@ <h1 class="translatable">
 if(window.location.host.includes('demandetesdonnees.fr') || window.location.host.includes('1315')) lang = 'fr';
 if(window.location.host.includes('pedidodedados.org') || window.location.host.includes('1316')) lang = 'pt';
 
+// Adds 'lang' attribute to <html>
+let htmlSelector = document.querySelector('html');
```
Comment (baltpeter) [inline static/404.html:85]: The region subtag is optional in the language tag (and it's not like we are targeting any specific regional language variant). Thus, we can do without the switch case. That's what I meant by a cleaner implementation. :) In fact, we can simplify the whole thing to one line:

```js
document.documentElement.lang = lang;
```

**Review by baltpeter — CHANGES_REQUESTED** [review]

**Comment by baltpeter** [comment]
As I mentioned, now that we have moved to another hosting solution, we don't need the hacky 404 implementation anymore. This PR is thus superseded by #398.

**Pull request closed without merging**